### PR TITLE
Added Bitdeli and GitHub Analytics Service

### PR DESCRIPTION
**Pull request by balzathor**
Added [Bitdeli](https://bitdeli.com) and [GitHub](https://github.com/blog/1672-introducing-github-traffic-analytics) Analytics service.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,9 @@ Table of Contents
   * https://keen.io/ - Up to 50,000 events/month free
   * http://www.inspectlet.com/ - 100 sessions/month free for 1 website
   * https://www.mousestats.com/ - 100 sessions/month free for 1 website
+  * https://bitdeli.com/ - Unlimited free analytics
+
+**P.S. - GitHub also has its own analytic service. [Read Here](https://github.com/blog/1672-introducing-github-traffic-analytics)**
```
Comment (ripienaar) [inline README.md]: I think I'd rather not have the PS in here, just list bitdeli please

Comment (balzathor) [inline README.md]: Removed PS line.

**Comment by ripienaar** [comment]
thanks!

**Comment by balzathor** [comment]
You're welcome :)

**Comment by ripienaar** [comment]
Sorry I backed that one out, this service is basically not functioning at present http://blog.bitdeli.com/post/77717727361/on-githubs-image-proxy

**Comment by balzathor** [comment]
Dang, never knew that. Thanks for pointing me towards the problem :)
and sorry for any inconvenience caused.

**Pull request merged**